src/Raven.Server/Config/Categories/PerformanceHintsConfiguration.cs

```diff
@@ -2,11 +2,16 @@
 using Raven.Server.Config.Attributes;
 using Raven.Server.Config.Settings;
 using Sparrow;
+using Sparrow.LowMemory;
 
 namespace Raven.Server.Config.Categories
 {
     public class PerformanceHintsConfiguration : ConfigurationCategory
     {
+        public PerformanceHintsConfiguration()
+        {
+            MinSwapSize = MemoryInformation.TotalPhysicalMemory;
```

Comment:
What about my comment regarding this - https://github.com/ravendb/ravendb/pull/11093#discussion_r485551261?